tests/test_annotations.py

```diff
@@ -8,13 +8,14 @@
 from tests import SCHEMA_FILE
 
 # Allowed annotation keys based on the provided whitelist
+# Note: Expected_value and Preferred_unit use GSC MIxS 6.2.0 capitalization
 ALLOWED_ANNOTATION_KEYS = {
-    "expected_value",
+    "Expected_value",
```

Comment:
Just for my own understanding, did MIxS adopt this capitalization standard? And then we're also changing some of _our own_ (not imported) annotations in `src/schema.core.yaml` match their standard?

Reply:
@turbomam check the rest of the NMDC code base for checks against those keys